Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+@objc class DailyForecast: NSObject {
+    let date: NSDate
+    let conditionsDescription: String
+    let highTemperature: Temperature
+    let lowTemperature: Temperature
+    
+    init(JSON: Dictionary<String, AnyObject>) {
```

Comment:
Sorry we're not more responsive. I appreciate you putting so much effort into this, and feel bad that we're blocking you. We don't actually work on this Mon - Thurs (we're on client work), so during the week we're limited to jumping in on our free time. We should be more responsive on Friday, which is when we have time to do non-client work.